docs/user_guide/network_visualization.rst

```diff
@@ -205,6 +205,45 @@ Or in a Jupyter Notebook:
 
 The substation diagrams will be arranged in a grid, based on the content of the matrix parameter. An empty string in the matrix will result in an empty spot in the grid.
 
+
+
```

Comment:
The section is very long, could you add a sub-section for customization with SldParameters, and one section for the customization with SldProfile?